rest/api.go

```diff
@@ -199,19 +211,19 @@ func (h *handler) handleGetDB() error {
 		lastSeq, _ = h.db.LastSequence()
 	}
 
-	response := db.Body{
-		"db_name":              h.db.Name,
-		"update_seq":           lastSeq,
-		"committed_update_seq": lastSeq,
-		"instance_start_time":  h.instanceStartTime(),
-		"compact_running":      h.db.IsCompactRunning(),
-		"purge_seq":            0, // TODO: Should track this value
-		"disk_format_version":  0, // Probably meaningless, but add for compatibility
-		"state":                runState,
+	var response = DatabaseRoot{
+		DBName:                        h.db.Name,
+		SequenceNumber:                lastSeq,
+		CommittedUpdateSequenceNumber: lastSeq,
+		InstanceStartName:             h.instanceStartTime(),
+		CompactRunning:                h.db.IsCompactRunning(),
+		PurgeSequenceNumber:           0,
```

Comment:
Maintain old comments

```suggestion
		PurgeSequenceNumber:           0, // TODO: Should track this value
```